asm/include/overlay_12_battle_controller.inc

```diff
@@ -7,7 +7,7 @@
 .public GetMonData
 .public GetPercentProgressTowardsNextLevel
 .public GetMonExpBySpeciesAndLevel
-.public sub_020708D8
+.public GetMonPicHeightBySpeciesGenderForme
```

Comment:
PokemonPicture_GetHeightBySpeciesGenderForm
(we are not using Forme, we already decided against it)